lib/Doctrine/ORM/Query/Expr.php

```diff
@@ -328,6 +328,19 @@ public function abs($x)
         return new Expr\Func('ABS', [$x]);
     }
 
+    /**
+     * Creates a MOD($x, $y) function expression to return the remainder of $x divided by $y.
+     *
+     * @param mixed $x
+     * @param mixed $y
```

Comment:
What types exactly can be passed here? Not that many I guess?

Reply:
I imagine it can be int|string|float|object (with _toString). It also follows current standard in file, I am not sure we should increase the scope of this PR and possibly document wrong types. Somebody should instead investigate types for all functions in file and document types in separate PR.